asf_tools/util.py

```diff
@@ -52,7 +52,10 @@ def tile_array(array: np.ndarray, tile_shape: Tuple[int, int] = (200, 200), pad_
     tile_list = []
     for rows in np.vsplit(array, range(tile_rows, array_rows, tile_rows)):
         tile_list.extend(np.hsplit(rows, range(tile_columns, array_columns, tile_columns)))
-    tiled = np.moveaxis(np.dstack(tile_list), -1, 0)
+
```

Comment:
Should the function type hints be updated with `array: Union[np.ndarray, np.ma.MaskedArray]`?

Also, not necessarily related to this PR, but the function definition doesn't have a type hint for the return value.

Reply:
Not needed as np.ndarray is the base class for np.ma.MaskedArray, but might make it clearer this works on masked arrays